src/audio/drc/drc_math_generic.c

```diff
@@ -177,7 +177,7 @@ inline int32_t drc_asin_fixed(int32_t x)
 }
 
 /*
- * Input x is Q6.26: (-32.0, 32.0)
+ * Input x is Q6.26; valid range: (0.0, 32.0); x <= 0 is not supported
```

Comment:
Sorry @ShriramShastry @cujomalainey I'm not following your previous comments, do we mean the range should be 1..31 here ? It just looks like the comment could be wrong ?

Reply:
Yes, Liam. We don't support 0 value while computing the logarithm. Johnny's fix will return back the function without processing the logarithm function if input value is 0. 
Comment section change was requested only to indicate valid supported range is  [1.0 to 32.0].

Reply:
Maybe it's a regional notation? But at least in NA ( represents non-inclusive range, where as [  an inclusive range.

So `(0, 32]` means 0 to 32 not including 0 but including 32. So 0.1 is in the rang with this notation.